docs/_specification/1.2-DRAFT/structure.md

```diff
@@ -71,84 +109,48 @@ The name of the _RO-Crate root_ directory is not defined, but a root directory i
-* The _RO-Crate Metadata Document_ MUST contain _RO-Crate JSON-LD_; a valid [JSON-LD 1.0] document in [flattened]  and [compacted] form
-* The _RO-Crate JSON-LD_ SHOULD use the _RO-Crate JSON-LD Context_ <https://w3id.org/ro/crate/1.2-DRAFT/context> by reference.
-
-
-[JSON-LD](https://json-ld.org/) is a structured form of [JSON] that can represent a _Linked Data_ graph. 
-
-A valid _RO-Crate JSON-LD_ graph MUST describe:
-
-1. The [RO-Crate Metadata Descriptor](root-data-entity#ro-crate-metadata-descriptor)
-2. The [Root Data Entity](root-data-entity#direct-properties-of-the-root-data-entity)
-3. Zero or more [Data Entities](data-entities)
-4. Zero or more [Contextual Entities](contextual-entities)
-
-It is RECOMMENDED that any referenced _contextual entities_ are also described in the _RO-Crate Metadata Document_ with the same identifier. Similarly it is RECOMMENDED that any _contextual entity_ in the _RO-Crate Metadata Document_ is linked to from at least one of the other entities using the same identifier. 
+The base RO-Crate specification makes no assumptions about the presence of any specific files or folders beyond the reserved RO-Crate files described above. 
 
-The appendix [RO-Crate JSON-LD](appendix/jsonld) details the general structure of the JSON-LD that is expected in the _RO-Crate Metadata Document_. In short, the rest of this specification describe the different types of entities that can be added as `{}` objects to the _RO-Crate JSON-LD_ `@graph` array below:
+Payload files may appear directly in the _RO-Crate Root_ alongside the _RO-Crate Metadata File_, and/or appear in sub-directories of the _RO-Crate Root_. Each file and directory MAY be represented as [Data Entities](data-entities) in the _RO-Crate Metadata File_.
 
-```json
-{ "@context": "https://w3id.org/ro/crate/1.2-DRAFT/context",
-  "@graph": [
+An RO-Crate may also contain [Web-based Data Entities](data-entities.html#web-based-data-entities) that are not present as part of the payload and referenced using absolute URIs. These may require additional preservation measures.
 
-  ]
-}
-```
+{% include callout.html type="tip" content="An RO-Crate [packaged with BagIt](appendix/implementation-notes#adding-ro-crate-to-bagit) may be [referencing external files](appendix/implementation-notes#referencing-external-files) which are not present in the _RO-Crate Root_ hierarchy until the BagIt has been _completed_. This method can be used for files that are large, require authentication or otherwise inconvenient to transfer with the RO-Crate, but which should nevertheless still be considered part of the _payload_.". RO-Crate 1.2 has a similar mechanism for referencing such external files using both an `@id` to reference a local path and [contentUrl]  property to indicate an source for that data. See the section on [File Data Entities](data-entities#file-data-entity) %}
```

Comment:
```suggestion
{% include callout.html type="tip" content="An RO-Crate [packaged with BagIt](appendix/implementation-notes#adding-ro-crate-to-bagit) may be [referencing external files](appendix/implementation-notes#referencing-external-files) which are not present in the _RO-Crate Root_ hierarchy until the BagIt has been _completed_. This method can be used for files that are large, require authentication or otherwise inconvenient to transfer with the RO-Crate, but which should nevertheless still be considered part of the _payload_.". RO-Crate has a similar mechanism for referencing such external files using both an `@id` to reference a local path and [contentUrl]  property to indicate a source for that data. See the section on [File Data Entities](data-entities#file-data-entity) %}
```